dao-erc20/src/erc20.rs

```diff
@@ -48,23 +48,23 @@ impl ERC20Interface for ERC20 {
     }
 
     fn name(&self) -> String {
-        self.name.get()
+        self.name.get_or_revert()
     }
 
     fn symbol(&self) -> String {
-        self.symbol.get()
+        self.symbol.get_or_revert()
     }
 
     fn decimals(&self) -> u8 {
-        self.decimals.get()
+        self.decimals.get_or_revert()
     }
 
     fn total_supply(&self) -> U256 {
-        self.total_supply.get()
+        self.total_supply.get_or_revert()
```

Comment:
Why to revert? I would return the default value.

Reply:
It shouldn't ever happen, as it is set during the init, so it really doesn't matter